bigquery/google/cloud/bigquery/client.py

```diff
@@ -38,6 +38,7 @@
 from google.cloud.bigquery.dataset import Dataset
 from google.cloud.bigquery.dataset import DatasetReference
 from google.cloud.bigquery.table import Table, _TABLE_HAS_NO_SCHEMA
+from google.cloud.bigquery.table import TableListItem
```

Comment:
While you're here, would you mind adding a "fix" (per the Google Python style guide) for the import above:

```python
from google.cloud.bigquery.table import Table
from google.cloud.bigquery.table import _TABLE_HAS_NO_SCHEMA
```

(Probably lexical order too?)

Reply:
Done.

Reply:
Looks like we'll have a bit of work cut out for us to get `flake8-import-order` happy once it works well for us.